Dockerfile

```diff
@@ -34,7 +34,7 @@
 # /////////////////////////////////////////////////////////////////////////////
 ARG DOCKER_ORG="usdotfhwastoldev"
 ARG DOCKER_TAG="develop"
-FROM ${DOCKER_ORG}/autoware.ai:${DOCKER_TAG} as base-image
+FROM ${DOCKER_ORG}/autoware.ai:tempest as base-image
```

Comment:
Is this change needed, or is it already covered by the change in `.github/workflows/Ci.yml`?

Reply:
yeah this is needed because it also overwrites any docker.yml dockerhub.yml functionality that passes in the DOCKER_TAG.
From those two workflows, branch's name will be passed in, which is wrong.